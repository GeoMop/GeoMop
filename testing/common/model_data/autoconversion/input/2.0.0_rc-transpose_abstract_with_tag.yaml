flow123d_version: 2.0.0_rc
problem: !Coupling_Sequential
  description: Test
  mesh:
    mesh_file: mesh.msh
  flow_equation: !Flow_Darcy_MH
  
    # Test data
    input_fields:
      # Transpose input_fields to array with two items
      rid: [1, 2]
      # and convert bc_flux values into records with given type
      bc_flux: !FieldFormula 
        value: [x+y, x-y]


    # Placeholder data
    nonlinear_solver:
      linear_solver: !Petsc
        r_tol: 1
        a_tol: 1
    balance: 
      balance_on: true
    output:
      output_stream: 
        file: ./test1.pvd
        format: !gmsh
      output_fields:
        - pressure_diff
