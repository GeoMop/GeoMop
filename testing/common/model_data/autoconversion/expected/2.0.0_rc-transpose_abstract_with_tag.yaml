flow123d_version: 2.0.0_rc
problem: !Coupling_Sequential
  description: Test
  mesh:
    mesh_file: mesh.msh
  flow_equation: !Flow_Darcy_MH
  
    # Test data
    input_fields:
      # Transposed input_fields to array with two items
      - rid: 1
        # and converted bc_flux value into record with given type
        bc_flux: !FieldFormula x+y
        
      - rid: 2
        bc_flux: !FieldFormula x-y


    # Placeholder data
    nonlinear_solver:
      linear_solver: !Petsc
        r_tol: 1
        a_tol: 1
    balance: 
      balance_on: true
    output:
      output_stream: 
        file: ./test1.pvd
        format: !gmsh
      output_fields:
        - pressure_diff
